modules/nw-egress-router-about.adoc

```diff
@@ -76,6 +111,9 @@ Specifically, ensure that the following are enabled:
 
 To avoid downtime, you can deploy an egress router pod with a `Deployment` resource, as in the following example. To create a new `Service` object for the example deployment, use the `oc expose deployment/egress-demo-controller` command.
 
+// mmckiern thinks the apiVersion s/b apps/v1
```

Comment:
Yes, I believe omitting the API group now emits a warning (or will in a future release).

Reply:
I'll revise on the next push.  There's always a next push.